variables.tf

```diff
@@ -77,4 +77,16 @@ variable "force_delete" {
   type        = bool
   description = "Whether to delete the repository even if it contains images"
   default     = false
-}
\ No newline at end of file
+}
+
+variable "enable_registry_replication" {
```

Comment:
Instead of using this additional variable, can you update to base our `count` for the replication resource off of the length of `replication_configuration_rules`?

For example: 

```
resource "aws_ecr_replication_configuration" "replication_configuration" {
  count = module.this.enabled && length(var.replication_configuration_rules) > 0 ? 1 : 0
```

Reply:
fixed